0007/README.md

```diff
@@ -0,0 +1,220 @@
+        year.
+    *   **Affiliation** / employer.
+*   **SC Members decide** if the individual meets the desirable properties
+    (described in the next section) and respond with an official determination
+    and rationale. Decisions are effective immediately.
+    *   SC Members should attempt to reach consensus. At a minimum:
+        *   SC Member applications require explicit approval from at least
+            50% of the existing SC Members, excluding the candidate.
+        *   Maintainer applications require explicit approval from at least 2
+            existing SC members (possibly including the candidate).
+*   Existing SC Members and Maintainers **annually re-apply** for the position,
+    using the same lightweight process above.
+    *   Purpose: to provide a natural way to remove members who are no longer
+        actively performing their duties.
+    *   Exception: If there are only three SC Members, they remain indefinitely
+        until another person applies.
+*   Applications and decisions are **recorded publicly**, such as through
+    GitHub Issues, for transparency and to provide examples for future
+    applicants.
+*   **OpenSSF may**, at their discretion and with due process and input from
+    the SCI WG:
+    *   appoint new SC Members to get up to the minimum three; or
+    *   remove an SC Member for cause.
+* **Appeals** may be made to the SC directly, and if needed, escalated to the
+    OpenSSF.
+
+### Clarify project roles
+
+*   Participant: **no change**, but clarify language (it’s a bit confusing as
+    written)
+*   Contributor: **no change**, but clarify language
+    *   A Maintainer may **delegate** some duties to a Contributor, such as
+        permission to triage issues or approve PRs
+*   Maintainer: an individual who is **trusted** and **expected to regularly
+    perform** day-to-day duties **for a specific sub-project** within the
+    slsa-framework organization.
+    *   A **Core Maintainer** is a maintainer of the slsa-framework/slsa repo.
+        This repository is special because it contains the core specification
+        and the overall project issue tracker.
+    *   A **Subproject Maintainer** is a maintainer of a different repo within
+        the slsa-framework organization.
+*   SC Member: a **senior leader** of the SLSA project who is **trusted** and
+    **expected** to ensure the health of the project overall, drive the overall
+    project strategy, and serve as an escalation point in the event of a
+    disagreement.
+    *   These duties are much less frequent than those of a Maintainer.
+    *   We generally **expect an SC Member to also be a Core Maintainer**, or a
+        Subproject Maintainer for a critical subproject, in order to ensure
+        that SC Members are well versed in the project.
+    *   Some Core Maintainers might not be SC Members. This may be a stepping
+        stone to becoming an SC Member.
```

Comment:
Suggestion from @brunodom was to soften this language. Maybe just say that:
- We expect SC Members to be well versed in the project (or to show expertise, or something like that)
- In practice SC Members are likely also Core Maintainers, but this is not a requirement.
- Remove the sentence about being a stepping stone.

Reply:
[aff2902](https://github.com/slsa-framework/slsa-proposals/pull/15/commits/aff29026d276b7dbb8659f75224ecf80cc3203ae) softens the language, as suggested.

@brunodom @MarkLodato PTAL.

Reply:
LGTM.